qa/tasks/cephfs/test_client_limits.py

```diff
@@ -179,6 +180,13 @@ def test_cap_acquisition_throttle_readdir(self):
         cap_acquisition_throttle_hit_count = self.perf_dump()['mds_server']['cap_acquisition_throttle']
         self.assertGreaterEqual(cap_acquisition_throttle_hit_count, 1)
 
+        # make sure that the throttle was reported in the events
+        ops_dump = self.fs.mds_asok(['dump_historic_ops'])
```

Comment:
```suggestion
        ops_dump = self.fs.rank_tell(['dump_historic_ops'])
```

Reply:
is it the same as communicating with the admin socket?

Reply:
I'm taking this suggestion. May I ask for the reasoning behind it?

Reply:
tell interfaces are recommended -- asok interface requires the command to be run locally from the node running the mds daemon. tell on the other hand, interfaces via the monitors and can be run from any node in the cluster.

Reply:
It also avoid a round-trip with the monitors: `mds_asok` has to figure out which MDS is rank 0 (via `fs dump`) and then send the command off to the node where the MDS is. `tell mds.X` on the other hand learns the MDSMap during setup of the `ceph` binary instance and sends off the command, all in one command. The latency difference may not be significant but it does help avoid cluttering the teuthology.log.